src/components/select-project/select-project.controller.ts

```diff
@@ -68,6 +70,16 @@ export class SelectProjectController implements angular.IController {
     return this.ctrl.projects && this.ctrl.selectedProject && !_.has(this.ctrl.selectedProject, 'metadata.uid');
   }
 
+  public groupChoicesBy = (item: any) => {
+    if (this.RecentlyViewedProjectsService.isRecentlyViewed(item.metadata.uid)) {
+      return "recently viewed";
```

Comment:
Do you mind making this "Recently Viewed" and "Other Projects" in case we ever change the grouping styles to remove the uppercase text transform?

Reply:
Ok, fixed